### ISSUE TITLE
Kernel local use-case fails due to 'network is unreachable'

### ISSUE THREAD

**Issue by denis-tingaikin** [networkservicemesh/cmd-forwarder-vpp#32]
## Forwarder vpp deployment

```
---
apiVersion: apps/v1
kind: DaemonSet
metadata:
  name: forwarder-vpp
  labels:
    app: forwarder-vpp
spec:
  selector:
    matchLabels:
      app: forwarder-vpp
  template:
    metadata:
      labels:
        app: forwarder-vpp
    spec:
      containers:
        - image: networkservicemesh/cmd-vpp-forwarder:latest
          imagePullPolicy: IfNotPresent
          name: forwarder-vpp
          securityContext:
            privileged: true
          env:
            - name: SPIFFE_ENDPOINT_SOCKET
              value: unix:///run/spire/sockets/agent.sock
            - name: NSM_TUNNEL_IP
              valueFrom:
                fieldRef:
                  fieldPath: status.podIP
            - name: NSM_CONNECT_TO
              value: unix:///var/lib/networkservicemesh/nsm.io.sock
            - name: NSM_NAME
              valueFrom:
                fieldRef:
                  fieldPath: metadata.name
          volumeMounts:
            - name: spire-agent-socket
              mountPath: /run/spire/sockets
              readOnly: true
            - name: nsm-socket
              mountPath: /var/lib/networkservicemesh
      volumes:
        - name: spire-agent-socket
          hostPath:
            path: /run/spire/sockets
            type: Directory
        - name: nsm-socket
          hostPath:
            path: /var/lib/networkservicemesh
            type: DirectoryOrCreate

```

## Steps to reproduce:

1. Use deployment that described above 

2. Follow these steps: https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/LocalConnection

## Logs
[logs.txt](https://github.com/networkservicemesh/cmd-forwarder-vpp/files/5841967/logs.txt)

**Comment by denis-tingaikin** [comment]
@edwarnicke Currently looking into this issue. Do you have any idea about this?

**Comment by edwarnicke** [comment]
Do you see the expected routes in the NSC Pod?

**Comment by edwarnicke** [comment]
I'm getting an ImagePullBackOff trying to get networkservicemesh/cmd-vpp-forwarder:latest

```
Events:
  Type     Reason     Age                From                         Message
  ----     ------     ----               ----                         -------
  Normal   Scheduled  77s                default-scheduler            Successfully assigned nsm-system/forwarder-vpp-289rj to kind-control-plane
  Normal   Pulling    39s (x3 over 77s)  kubelet, kind-control-plane  Pulling image "networkservicemesh/cmd-vpp-forwarder:latest"
  Warning  Failed     38s (x3 over 76s)  kubelet, kind-control-plane  Failed to pull image "networkservicemesh/cmd-vpp-forwarder:latest": rpc error: code = Unknown desc = failed to resolve image "docker.io/networkservicemesh/cmd-vpp-forwarder:latest": no available registry endpoint: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
  Warning  Failed     38s (x3 over 76s)  kubelet, kind-control-plane  Error: ErrImagePull
  Normal   BackOff    12s (x4 over 75s)  kubelet, kind-control-plane  Back-off pulling image "networkservicemesh/cmd-vpp-forwarder:latest"
  Warning  Failed     12s (x4 over 75s)  kubelet, kind-control-plane  Error: ImagePullBackOff
```

**Comment by denis-tingaikin** [comment]
> Do you see the expected routes in the NSC Pod?

I also thought about this, currently looking into https://github.com/networkservicemesh/cmd-nse-icmp-responder, looks like routes are missed there.

> I'm getting an ImagePullBackOff trying to get networkservicemesh/cmd-vpp-forwarder:latest

I built it manually with the changes from https://github.com/networkservicemesh/cmd-forwarder-vpp/pull/33

**Comment by edwarnicke** [comment]
I think this has been resolved by https://github.com/networkservicemesh/cmd-forwarder-vpp/pull/31 carrying the fix https://github.com/networkservicemesh/sdk-vpp/pull/62

**Comment by denis-tingaikin** [comment]
Fixed by #31

@edwarnicke Thanks!
